requirements.in

```diff
@@ -1,4 +1,5 @@
 aiobotocore
 aiohttp
+aiohttp_cors
```

Comment:
This also needs to be compiled into the [`requirements.txt`](https://github.com/pypa/conveyor/blob/master/requirements.txt) file.